examples/with-api-destination/main.tf

```diff
@@ -0,0 +1,68 @@
+provider "aws" {
+  region = "ap-southeast-1"
+
+  # Make it faster by skipping something
+  skip_get_ec2_platforms      = true
+  skip_metadata_api_check     = true
+  skip_region_validation      = true
+  skip_credentials_validation = true
+  skip_requesting_account_id  = true
+}
+
+module "eventbridge" {
+  source = "../../"
+
+  create_bus              = true
+  create_connections      = true
+  create_api_destinations = true
+
+  bus_name = "${random_pet.this.id}-bus"
+
+  connections = {
+    "requestbin" = {
+      authorization_type = "BASIC"
+      auth_parameters = {
+        basic = {
+          username = random_pet.this.id
+          password = random_pet.this.id
+        }
+      }
+    }
+    "github" = {
+      authorization_type = "API_KEY"
+      auth_parameters = {
+        api_key = {
+          key   = "x-signature-id"
+          value = random_pet.this.id
+        }
+      }
+    }
+  }
+
+  api_destinations = {
+    "requestbin" = {
+      description                      = "my requestbin endpoint"
+      invocation_endpoint              = "https://pipedream.com/@svenlito/test-rb-1-p_rvCQGl1"
```

Comment:
Let's use something publicly available. I usually use this service - https://smee.io/